st2common/st2common/models/db/__init__.py

```diff
@@ -87,6 +88,18 @@ def db_setup(db_name, db_host, db_port, username=None, password=None, ensure_ind
                                                 port=db_port, tz_aware=True,
                                                 username=username, password=password,
                                                 **ssl_kwargs)
+    return connection
```

Comment:
This connection object is never used anywhere else. Shouldn't you be disconnecting somewhere?

Reply:
I was modeling this after the already existing `db_setup()` method (just below this one).

I believe the connection is disconnected in `st2common/st2common/persistence/cleanup.py` `teardown()`

If you would like me to change this behavior i'm fine with it, please advise on what you'd like to see.

Reply:
Okay, I see how the original db_setup returns a connection. Ignore my comment.